Guides/UPI/baremetal/node-hostname-resolution.md

```diff
@@ -1,6 +1,142 @@
 Resolving the Node Hostname
 ---
 
+# Background
 Nodes require a correctly configured hostname in order to join and remain a fully functional cluster member. The hostname is used to generate certificates and also figure out what to call itself.
 
-TBC..!
\ No newline at end of file
+The OKD installation docs mention that setting valid PTR records will allow the system to automatically detect the hostname. This does not seem to conistently be the case. In addition you may be in an environment where you are unable to edit PTR records.
```

Comment:
```suggestion
The OKD installation docs mention that setting valid PTR records will allow the system to automatically detect the hostname. This does not seem to consistently be the case. In addition you may be in an environment where you are unable to edit PTR records.
```
I think its been a NetworkManager bug in OKD 4.6 from december. It shouldn't be the case anymore, so probably not worth mentioning

Reply:
I observed this bug in the second and third week of January during cluster deploys, if it's been fixed since then I wouldn't have noticed since I've been setting it manually via the MachineConfig.

Reply:
Hmm, okay, lets file a bug on OKD (most likely there's already one) and refer to it

Reply:
I'll see if it's still reproducible on the latest version of OKD and file a bug if not. The related issue I looked at was #394 which is now closed - but most of that conversation was around vSphere.